script.js

```diff
@@ -1,4 +1,523 @@
+  '<img src="https://c.tenor.com/O5exqlsLA0AAAAAi/gabby-cute.gif"/>';
+var rollImage =
+  '<img src="https://c.tenor.com/39SseKZP8jYAAAAi/quby-dance.gif"/>';
+var bombImage = '<img src="https://c.tenor.com/8C3lFqyUiroAAAAi/quby.gif"/>';
+var standImage =
+  '<img src="https://c.tenor.com/ivBgGj-cy80AAAAj/tkthao219-quby.gif"/>';
+var finishImage =
+  '<img src="https://c.tenor.com/5YF790yqQ1UAAAAi/fbcuteboy.gif"/>';
+var gameOverImage =
+  '<img src="https://c.tenor.com/wAx24E9mC0MAAAAi/t%E1%BB%A9c-smile.gif"/>';
+var makeDeck = function () {
+  var cardDeck = [];
+  var suits = ["hearts", "diamonds", "clubs", "spades"];
+  var suitsEmoji = ["♥︎", "♦︎", "♣︎", "♠︎"];
+  var suitIndex = 0;
+  while (suitIndex < suits.length) {
+    var currentSuit = suits[suitIndex];
+    var currentEmoji = suitsEmoji[suitIndex];
+    var rankCounter = 1;
+    while (rankCounter <= 13) {
+      var cardName = rankCounter;
+      if (cardName == 1) {
+        cardName = "ace";
+      } else if (cardName == 11) {
+        cardName = "jack";
+      } else if (cardName == 12) {
+        cardName = "queen";
+      } else if (cardName == 13) {
+        cardName = "king";
+      }
+
+      var currentValue = rankCounter;
+      if (currentValue == 11 || currentValue == 12 || currentValue == 13) {
+        currentValue = 10;
+      }
+
+      var card = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankCounter,
+        emoji: currentEmoji,
+        blackjackValue: currentValue,
+      };
+      cardDeck.push(card);
+      rankCounter += 1;
+    }
+    suitIndex += 1;
+  }
+  return cardDeck;
+};
+
+var getRandomIndex = function (max) {
+  return Math.floor(Math.random() * max);
+};
+
+var shuffleCards = function (cardDeck) {
+  var currentIndex = 0;
+  while (currentIndex < cardDeck.length) {
+    var randomIndex = getRandomIndex(cardDeck.length);
+    var randomCard = cardDeck[randomIndex];
+    var currentCard = cardDeck[currentIndex];
+    cardDeck[currentIndex] = randomCard;
+    cardDeck[randomIndex] = currentCard;
+    currentIndex = currentIndex + 1;
+  }
+  return cardDeck;
+};
+
+var displayCards = function (player) {
+  var message = "";
+  if (player == numOfPlayers) {
+    player = "dealer";
+  }
+  for (var i = 0; i < master[player].decks.length; i += 1) {
+    message += `<br>${master[player].decks[i].name} of ${master[player].decks[i].emoji}`;
+  }
+  return message;
+};
+
+var displayOneDealerCard = function () {
+  var message = `<br><br>❗️One of the Dealer's Cards is ${master["dealer"].decks[0].name} of ${master["dealer"].decks[0].emoji}❗️`;
+  return message;
+};
+
+var displaySplitCards = function (player) {
+  var message = "";
+  master[player].blackjackValues = [];
+  for (var i = 0; i < master[player].decks.length; i += 1) {
+    var aceCounter = 0;
+    var total = 0;
+    var handNum = i + 1;
+    message += `<br><b>Hand ${handNum}</b>:`;
+    for (var j = 0; j < master[player].decks[i].length; j += 1) {
```

Comment:
master[player].decks
master[player].decks[i]
can be assigned to playerDeck and playerCard, any manipulation to these variables will be fine as these are reference variables